tide_site_restriction.module

```diff
@@ -133,6 +137,14 @@ function tide_site_restriction_views_pre_view(ViewExecutable $view, $display_id,
       }
     }
   }
+  // Excluding "Delete Content" option for editors SDPA-3781.
```

Comment:
@vincent-gao  there is no specific way to hide a filter value for a specific role, so that is why I did it in that place, the same way the assigned site gets loaded in the subsite filter in this view.